### PR TITLE
Set a more pertinant working directory

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ ENV TILEGENERATION_CONFIGFILE=/etc/tilegeneration/config.yaml \
 
 EXPOSE 80
 
-WORKDIR /project
+WORKDIR /etc/tilegeneration/
```
Comment (pvalsecc) [inline Dockerfile]: We will have to fix all the projects using that image...

File: tilecloud_chain/server.py [hunk 2]
```diff
@@ -440,7 +440,11 @@ def response(data, headers=None, **kwargs):
         return [data]
 
 
-def app_factory(global_config, configfile='tilegeneration/config.yaml', **local_conf):
+def app_factory(
+    global_config,
```
Comment (pvalsecc) [inline tilecloud_chain/server.py]: I'm no fan of this indentation!

File: Dockerfile (hunk 1 above)
Comment (sbrunner) [reply Dockerfile]: Why ?
This didn't have any incidence ...

File: tilecloud_chain/server.py (hunk 2 above)
Comment (sbrunner) [reply tilecloud_chain/server.py]: I don't found better one ...

**Pull request merged**